src/pages/OnboardingEmployees/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {View} from 'react-native';
+import FocusTrapForScreens from '@components/FocusTrap/FocusTrapForScreen';
+import useThemeStyles from '@hooks/useThemeStyles';
+import BaseOnboardingEmployees from './BaseOnboardingEmployees';
+import type {OnboardingEmployeesProps} from './types';
+
+function OnboardingEmployees({...rest}: OnboardingEmployeesProps) {
```

Comment:
```suggestion
function OnboardingEmployees(props: OnboardingEmployeesProps) {
```
No need to destruct props twice. The use of `rest` is wrong here as we are not taking any other props (rest === all)